fms/models/__init__.py

```diff
@@ -243,7 +241,10 @@ def get_model(
         if world_size > 1:
             distributed_strategy = "tp"
 
-    device = torch.device(device_type, local_rank)
+    if device_type == "cuda":
+        device = torch.device(device_type, local_rank)
+    else:
+        device = torch.device(device_type)
```

Comment:
I think local-rank is 0 by default and `torch.device('cpu', 0)` works, so I don't think we actually need the condition here

Reply:
I think this failed for me at some point, that's why I added it

Reply:
the inverse of the above comment :-)